src/client/client.h

```diff
@@ -437,6 +437,15 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	{
 		return m_env.getLocalPlayer()->formspec_prepend;
 	}
+	inline u16 getMeshChunk() {
+		return m_mesh_chunk;
+	}
+	inline s16 getMeshPos(s16 p) {
+		return ((p - (p < 0) * (m_mesh_chunk - 1)) / m_mesh_chunk * m_mesh_chunk);
+	}
+	inline bool checkMeshPos(v3s16 &p) {
+		return ((p.X + p.Y + p.X) % m_mesh_chunk) == 0;
```

Comment:
p.X + p.Y + p.Z

Reply:
I keep messing that one up. I'm slightly dyslexic I think :(